two_encoder_module/two_encoder_src/two_encoder_src.ino

```diff
@@ -288,8 +293,8 @@ void initEncoder(struct Encoder* encoder, int32_t pinA, int32_t pinB, int32_t bu
 int8_t updateEncoder(struct Encoder* encoder)
 {
     int8_t encoderMotion = 0;
-    int pinACurrent = digitalRead(encoder->pinA);
-    int pinBCurrent = digitalRead(encoder->pinB);
+    uint8_t pinACurrent = digitalRead(encoder->pinA);
+    uint8_t pinBCurrent = digitalRead(encoder->pinB);
```

Comment:
The digitalRead functions return int, so they should be stored in variables of type int. Request that variables representing the pin state be changed back to int. The rest of the type changes are fine.